otherlibs/stdune/src/map.ml

```diff
@@ -204,7 +204,9 @@ module Make (Key : Key) : S with type key = Key.t = struct
 
   let filter_opt t = filter_map t ~f:Fun.id
 
-  let superpose a b = union a b ~f:(fun _ _ y -> Some y)
+  let superpose =
+    let f _ _ y = Some y in
```

Comment:
Why is `superpose` right biased? It seems surprising to me, and the call sites you're replacing were left-biased.

Can we make `superpose` left-biased?

Reply:
> Why is superpose right-biased?

No idea, I suppose it was Jeremie's decision early on.

> and the call sites you're replacing were left-biased

I flipped the arguments in the applications though.

> Can we make superpose left-biased?

Why not, I don't mind doing it in a separate PR.  Both choices seem a little arbitrary to me and I would prefer some labels to make this clear.

Reply:
> I flipped the arguments in the applications though.

Oh, I didn't notice that. I think my preference would be to first make `superimpose` left-biased and undo the swapping in this PR.

Left-biased operations are a lot more common, I think, which is why I find the current behaviour of `superimpose` surprising (and it's probably just accidental).